src/pageload.js

```diff
@@ -0,0 +1,68 @@
+import leadImage from './assets/pexels-cassiano-psomas-1045199.jpg';
+import firstImage from './assets/pexels-dhyamis-kleber-3584317.jpg';
+import fifthImage from './assets/pexels-tookapic-7782.jpg';
+import thirdImage from './assets/pexels-chevanon-photography-312418.jpg';
+
+export function pageload() {
+    const contentDiv = document.querySelector('#content');
+
+    contentDiv.replaceChildren();
+
+    const leadCont = document.createElement("div");
+    leadCont.classList.add("lead-container")
+    const leadImg = document.createElement("img");
+    leadImg.classList.add("lead-img");
+    leadImg.src = leadImage;
+    leadImg.alt = 'Lead Img';
+    leadCont.append(leadImg);
+    contentDiv.appendChild(leadCont);
+
+    const firstSection = document.createElement("div");
+    firstSection.classList.add("first-section-paragraph");
+    firstSection.textContent = "Cassual Restaurant, combined with the prominent page, gives the brand a modern, forward-thinking feel, and this is a great investment for a business aiming at future rapid growth. However, this website probably cost a fair chunk to build - so this route certainly isnt for everyone.";
+    contentDiv.appendChild(firstSection);
+
+    const headinf = document.createElement("h1");
+    headinf.classList.add("headdinf");
+    headinf.textContent = "Dishes Available";
+    contentDiv.appendChild(headinf);
+
+    const dishesContainer = document.createElement("div");
+    dishesContainer.classList.add("dishescont");
+
+    const firstDish = document.createElement("div");
+    firstDish.classList.add("firstDish");
+    const firstImg = document.createElement("img");
+    firstImg.classList.add("first-dish-imag");
+    firstImg.src = firstImage;
+    firstDish.textContent = "Black Tea as low as \n $2";
+    firstImg.alt = "first dish Image";
+    
+    firstDish.append(firstImg);
+    dishesContainer.append(firstDish);
+
+    const fifthDish = document.createElement("div");
+    fifthDish.classList.add("firstDish");
+    const fifthImg = document.createElement("img");
+    fifthImg.classList.add("first-dish-imag");
+    fifthImg.src = fifthImage;
+    fifthDish.textContent = "Stake with Chips as low as \n $7";
+    fifthImg.alt = "fifth dish Image";
+    
+    fifthDish.append(fifthImg);
+    dishesContainer.append(fifthDish);
+
+    const thirdDish = document.createElement("div");
+    thirdDish.classList.add("firstDish");
+    const thirdImg = document.createElement("img");
+    thirdImg.classList.add("first-dish-imag");
```

Comment:
- [ ] The pictures here should be aligned also 
![image](https://user-images.githubusercontent.com/86371332/174452949-8e5275fc-888c-4d75-a8bb-ccf81c5e810c.png)

Reply:
Noted